src/extension.ts

```diff
@@ -279,6 +287,9 @@ export async function activate(context: ExtensionContext) {
         commands.registerCommand('r.runCommandWithEditorPath', runCommandWithEditorPath),
         commands.registerCommand('r.runCommand', runCommand),
         commands.registerCommand('r.launchAddinPicker', launchAddinPicker),
+        commands.registerCommand('r.workspaceViewer.refreshEntry', () => rWorkspace.refresh()),
+        commands.registerCommand('r.workspaceViewer.view', (node: WorkspaceItem) => runTextInTerm(`View(${node.label})`)),
+        commands.registerCommand('r.workspaceViewer.clear', () => runTextInTerm(`rm(list = ls())`)),
```

Comment:
Do we ignore those symbols starting with `.`? Or we should send `rm(list = ls(all.names = TRUE))`?

Reply:
I think ideally we would give the users the choice, either through a settings option or a dialog box?

Reply:
Removing all variables is quite destructive. I guess we should show a dialog box for the user to confirm.

As for whether to remove all variables (including hidden variables), the RStudio behavior is following

![image](https://user-images.githubusercontent.com/4662568/100567700-1202ef00-3304-11eb-894c-35ba220d9eae.png)

Could we do something similar? Not sure if we could put a checkbox on the dialog.

Reply:
I've implemented a message box: 

![image](https://user-images.githubusercontent.com/60372411/100695172-94cd8d80-3388-11eb-88c0-26be019dde85.png)

There doesn't seem to be an option to use a checkbox as per https://github.com/Microsoft/vscode/issues/56722. I think we could have two confirm buttons, or rely on the settings menu?

Reply:
Implemented a settings option in 8c86577

![image](https://user-images.githubusercontent.com/60372411/100949656-a175f100-3502-11eb-83a8-099a831bfcb3.png)

Reply:
LGTM